ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -73,7 +73,7 @@ Item {
         }
 
         function markAllMessagesReadIfMostRecentMessageIsInViewport() {
-            if (!isMostRecentMessageInViewport || !chatLogView.visible || keepUnread) {
+            if (!Qt.application.active || !isMostRecentMessageInViewport || !chatLogView.visible || keepUnread) {
```

Comment:
```suggestion
            if (Qt.application.state != Qt.ApplicationActive || !isMostRecentMessageInViewport || !chatLogView.visible || keepUnread) {
```

Reply:
`Qt.application.active` is deprecated (https://doc.qt.io/qt-5/qml-qtqml-qt.html#application-prop)

Reply:
I blame copilot :see_no_evil: